src/utils/file_library.hpp

```diff
@@ -0,0 +1,57 @@
+/*************************************************************************
+ * Copyright (C) 2018-2020 Blue Brain Project
+ *
+ * This file is part of NMODL distributed under the terms of the GNU
+ * Lesser General Public License. See top-level LICENSE file for details.
+ *************************************************************************/
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+namespace nmodl {
+
```

Comment:
no doxygen documentation?

Reply:
Fixed in d26e5ef676db7da848eff84ac49ed90efa87538f